backend/huggingface/Chroma/scheduler/scheduler_config.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
copy&paste from flux file, same as for all the files bellow where I don't put notes